sdk/python/featurestore_sample/notebooks/sdk_only/4. Enable online store and run online inference.ipynb

```diff
@@ -594,9 +588,10 @@
     "    version=\"1\",\n",
     "    feature_window_start_time=st,\n",
     "    feature_window_end_time=ed,\n",
-    "    data_status=[\"None\"],\n",
+    "    # data_status=[\"None\"],\n",
     ")\n",
-    "print(poller.result().job_ids)"
+    "# print(poller.result().job_ids)\n",
```

Comment:
Why do we have this in comments? Is this notebook for GA?

Reply:
This will be commented out on 11/13 once the GA release version of the notebook is available.